src/pages/layer-2/index.tsx

```diff
@@ -0,0 +1,630 @@
+              }}
+            />
+          </div>
+        </div>
+      </div>
+
+      {/* TODO: Setup for translation */}
+      <div id="layer-2-stats-box" className="w-full px-8 py-9">
+        <div className="m-auto max-w-[992px] py-9">
+          <div className="flex flex-col gap-8 border border-body-light p-8 md:flex-row md:gap-14">
+            <div className="flex-1">
+              <div className="max-w-[224px]">
+                <p className="text-5xl">
+                  $
+                  {growThePieData.dailyTxCosts["ethereum"].toLocaleString(
+                    locale as Lang,
+                    { minimumFractionDigits: 2, maximumFractionDigits: 2 }
+                  )}
+                </p>
+                <p className="text-body-medium">
+                  Average transaction cost on the Ethereum blockchain
+                </p>
+              </div>
+            </div>
+            <div className="h-auto w-auto self-stretch border-b border-body-light md:border-r" />
+            <div className="flex-1">
+              <div className="max-w-[224px]">
+                <p className="text-5xl">
+                  $
+                  {medianTxCost.toLocaleString(locale as Lang, {
+                    minimumFractionDigits: 2,
+                    maximumFractionDigits: 3,
+                  })}
+                </p>
+                <p className="text-body-medium">
+                  Average transaction cost on Ethereum backed networks
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+        {/* TODO: Commented out for now, remove before release if we don't ship with it. */}
+        {/* <div className="relative m-auto -mt-4 h-44 w-44 -rotate-12 rounded-full bg-primary sm:absolute sm:right-10 sm:-mt-28 md:-mt-80 xl:right-52">
+          <div className="flex h-full w-full flex-col items-center justify-center">
+            <p className="text-background">over</p>
+            <p className="text-5xl text-background">100x</p>
+            <p className="text-background">cheaper</p>
+          </div>
+        </div> */}
+      </div>
+
+      {/* TODO: Setup for translation */}
+      <div id="layer-2-the-network-of-networks" className="w-full py-9 sm:px-8">
+        <div className="flex flex-col gap-8 overflow-hidden bg-[#B9B9F1] bg-opacity-20 px-4 py-10 text-center">
+          <h2>The network of networks</h2>
+          <p className="font-md">
+            Ethereum&apos;s strength and security provides a platform for other
+            networks to build upon. With a single account, everything is
+            compatible and connects seamlessly.
+          </p>
+          <div className="relative m-auto h-[275px] w-[275px] sm:h-[375px] sm:w-[375px]">
+            {/* Outer ring */}
+            <div className="absolute inset-0 rounded-full border border-dashed border-body-medium"></div>
+            <div className="absolute inset-0 animate-spin-30 rounded-full">
```

Comment:
<img width="420" alt="image" src="https://github.com/user-attachments/assets/b86cb9c4-4cf2-46f3-a466-c2ed82c3ff51">
<img width="411" alt="image" src="https://github.com/user-attachments/assets/8b8baf96-1344-4205-9ee0-ce8136cff4bf">

Just noting some of these are hard to see in light mode, and others in dark. Perhaps we force a circular background color here?

Reply:
Updated the logos and rounded effect